lifecycle-operator/chart/doc.yaml

```diff
@@ -0,0 +1,121 @@
+# yamllint disable rule:line-length
```

Comment:
i think the sections here could be re-organized to start with the lifecycle operator and then have the scheduler things afterwards